src/v2/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -127,17 +127,18 @@ export const SavedSearchAlertModal: React.FC<SavedSearchAlertFormProps> = ({
 
   return (
     <FormikProvider value={formik}>
-      <Form onSubmit={formik.handleSubmit} role="form">
-        <Modal
-          show={visible}
+      {visible && (
```

Comment:
hm wait do we still need the `<Form>` here? Or does it just work because it's wrapped in `<FormikProvider>` and has the `formik.handleSubmit()` call?

Reply:
> Or does it just work because it's wrapped in <FormikProvider> and has the formik.handleSubmit() call

Yes, you are absolutely right. We don't need to use the `Form` component, since we manually call `formik.handleSubmit`